packages/main/src/RPA/Robocorp/WorkItems.py

```diff
@@ -288,21 +289,27 @@ class FileAdapter(BaseAdapter):
 
     Reads inputs from the given database file, and writes
     all created output items into an adjacent file
-    with the suffix ``<filename>.output.json``.
+    with the suffix ``<filename>.output.json``. If the output path is provided by an
+    env var explicitly, then the file will be saved with the provided path and name.
 
     Reads and writes all work item files from/to the same parent
     folder as the given input database.
 
     Required environment variables:
 
-    * RPA_WORKITEMS_PATH:   Path to work items database file
+    * RPA_INPUT_WORKITEM_PATH:   Path to work items database file
```

Comment:
How about specifying the `RPA_OUTPUT_WORKITEM_PATH ` too, but this time under a new section of "Optional environment variables:" so we can link this doc in the `deprecation()` warnings.